### PR TITLE
v2.10.0 - cuisinesWidget component is added to the docs

### PR DESCRIPTION
**Pull request by dandel10n**
### Added
- `c-cuisinesWidget` component

### Changed
- `fozzie` module version update

screen shot 2018-12-04 at 11 15 12

Without fozzie docs grid:
screen shot 2018-12-04 at 11 17 39

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
- [x] This code has been checked with regard to our accessibility standards

## Browsers Tested

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] IE 11
- [x] Edge
- [x] Mobile (i.e. iPhone/Android - via browser responsive mode)

### CODE REVIEW

File: assets/src/scss/components/_demoBlock.scss [hunk 1]
```diff
@@ -46,6 +46,10 @@
         }
     }
 
+        .demo-visual--spacingTopAndBottom {
```
[the remark this answers is not in the document]
Comment (dandel10n) [reply assets/src/scss/components/_demoBlock.scss]: ✅ Improved

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Added
+- `c-cuisinesWidget` component
+
+### Changed
+- `fozzie` module version update
+
+
 v2.9.0
 ------------------------------
 *November 28, 2018*
```
Comment (ashleynolan) [inline CHANGELOG.md]: Should remove the rounded corners from the `cuisines-widget.jpg`.  Guessing this should be handled by the CSS?

Comment (dandel10n) [reply CHANGELOG.md]: @ashleynolan  Original assets for this block in the design are with rounded corners. Should I ask Bruno to upload them without them?

Comment (dandel10n) [reply CHANGELOG.md]: I've uploaded version with straight corners(just cut the original image a little bit)

Comment (ashleynolan) [reply CHANGELOG.md]: 👍

**Pull request merged**